documentation-website/cypress/e2e/language-switch.cy.ts

```diff
@@ -21,7 +21,7 @@ describe('language-switch', () => {
   },);
 
   it('changes language correctly', () => {
-    const pageSignature = '@idrinth/api-bench';
+    const pageSignature = '@idrinth-api-bench/api-bench';
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted